app/uk/gov/nationalarchives/omega/editorial/models/package.scala

```diff
@@ -83,10 +83,190 @@ package object editSetRecords {
     .validate[EditSetRecord]
     .get
 
+  val editSetRecord4: EditSetRecord = Json
+    .parse(
+      """{
+        |  "ccr" : "COAL 80/80/1",
```

Comment:
Can we please increment the CCRs as well as the OCIs so that they appear as distinct records.